### DISCUSSION TITLE
how to change the categories style in a listing?

### DISCUSSION THREAD

**Discussion by ligea-alexander** [quarto-dev/quarto-cli#9883] [Q&A] [answered]
### Description

After

----
listing:
categories: true
-----

how do you make use of the styling options to change appearance that are specified in the documentation? i.e. 'unnumbered', 'numbered', 'cloud'?

https://quarto.org/docs/websites/website-listings.html#category-appearance

**Comment by mcanouil** [comment, marked as answer]
set the value to what you want:

```yaml
listing:
  categories: cloud
```

**Reply by ligea-alexander** [reply]
Thanks!
